src/main/java/io/strimzi/kafka/bridge/Handler.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge;
+
+/**
+ * Represents a functional interface for handling callback when an asynchronous operation ends
+ *
+ * @param <T> type of the data to be handled
+ */
+public interface Handler<T> {
```

Comment:
If it's intended to be a functional interface then you should add `@FunctionalInterface`.